README.md

```diff
@@ -34,7 +34,7 @@ jobs:
         id: get-changed-files
         run: |
           CHANGED_FILES=$(git diff \
-            --name-only ${{ github.event.before }} ${{ github.sha }} \
+            --name-only --diff-filter=ACMRT ${{ github.event.before }} ${{ github.sha }} \
```

Comment:
lower case `d` means the inverse `D` (enable everything but deletes)

Reply:
So you are suggesting I could do
```suggestion
            --name-only --diff-filter=d ${{ github.event.before }} ${{ github.sha }} \
```